services/db/migrations/019_cvm.up.sql

```diff
@@ -0,0 +1,29 @@
+create table `cvm_block`
```

Comment:
It looks like we're not writing to this table. Is that an oversight or should we remove it?

Reply:
The cvm_block table is written to by the producer.  It is a mechanism for persisting the block numbers we have processed from avalanchego.

We don't want to start up and re-read all the blocks from the node, just the newer ones we have not seen.